examples/hmac.rs

```diff
@@ -0,0 +1,64 @@
+#![no_std]
+
+use core::fmt::Write;
+use libtock::debug::print_as_hex;
+use libtock::hmac::{HmacDataBuffer, HmacDestBuffer, HmacKeyBuffer};
+use libtock::result::TockResult;
+use libtock::syscalls;
+
+#[libtock::main]
+async fn main() -> TockResult<()> {
+    let mut drivers = libtock::retrieve_drivers()?;
+
+    let mut console = drivers.console.create_console();
+
+    writeln!(console, "Starting HMAC example")?;
```

Comment:
It feels like there are too many blank lines in `main()`. Perhaps the first 4 lines (which currently have blank lines between them) should just be merged into a 4-line block?